pkg/tsdb/elasticsearch/time_series_query.go

```diff
@@ -57,137 +57,25 @@ func (e *timeSeriesQuery) execute() (*backend.QueryDataResponse, error) {
 }
 
 func (e *timeSeriesQuery) processQuery(q *Query, ms *es.MultiSearchRequestBuilder, from, to int64) error {
-	defaultTimeField := e.client.GetTimeField()
+	err := isQueryWithError(q)
```

Comment:
This is the same as previously, we just check it at the beginning, because if the query has error, we don't need to create search query at all. 
```
if len(q.Metrics) == 0 || !(q.Metrics[0].Type == rawDataType || q.Metrics[0].Type == rawDocumentType || q.Metrics[0].Type == logsType) {
			return fmt.Errorf("invalid query, missing metrics and aggregations")
		}
```